Fix changing log level at runtime in native image. Fixes #5083

auke- opened the pull request:
The log levels can be adjusted at runtime using the loggers endpoint.
This doesn't work when the application is build as a GraalVM native
image, because the LogLevel class is missing the reflection
configuration.

By marking the LogLevel class as Introspected the correct reflection
configuration is generated and the loggers endpoint can be used in a
native image.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/micronaut-projects/micronaut-core?pullRequest=5105) <br/>All committers have signed the CLA.

auke- committed:
Fix changing log level at runtime in native image. Fixes #5083

The log levels can be adjusted at runtime using the loggers endpoint.
This doesn't work when the application is build as a GraalVM native
image, because the LogLevel class is missing the reflection
configuration.

By marking the LogLevel class as Introspected the correct reflection
configuration is generated and the loggers endpoint can be used in a
native image.

jameskleeh commented:
Thanks!
